utils/constants.tsx

```diff
@@ -1872,6 +1872,7 @@ export const Constants = {
     TRIAL_MODAL_AUTO_SHOWN: 'trial_modal_auto_shown',
     DEFAULT_SITE_URL: 'http://localhost:8065',
     CHANNEL_HEADER_BUTTON_DISABLE_TIMEOUT: 1000,
+    TIMESTAMP_FOR_DEFAULT_DM_NUMBER: 1663186000,
```

Comment:
It is referring to Wednesday, September 14, 2022. I think best would be to change this date to the publishing date of the version which this change is included.